fr_FR/LC_MESSAGES/search.po

```diff
@@ -29,17 +29,17 @@ msgid "RESULTS_FOUND_FORMAT_MULTIPLE"
 msgstr "%1$d résultats trouvés pour \"%2$s\""
 
 msgid "NO_RESULTS_FOUND"
-msgstr "Pas de résultat trouvé"
+msgstr "Aucun résultat trouvé"
 
 msgid "SEARCH_SHORT_KEY"
-msgstr "Flipnote ID"
+msgstr "@ID Flipnote"
 
 msgid "SHORT_KEY_NO_RESULTS"
-msgstr "No Flipnote found"
+msgstr "Aucune Flipnote trouvée"
 
 msgid "SHORT_KEY_FORMAT_GUIDE"
-msgstr "Not finding what you're looking for? You may have entered the Flipnote ID incorrectly. Flipnote IDs are six digits long and can be found in the details section for the Flipnote."
+msgstr "Vous ne trouvez pas ce que vous cherchez ? Vous avez peut-être mal écrit l'ID de la Flipnote. Les ID Flipnotes contiennent 6 chiffres et peuvent être trouvées dans les détails de la Flipnote."
```

Comment:
There's a space after 'cherchez' -- can you remove that and push the change in one more commit?

Reply:
In French there's always a space before "!" and "?"